_posts/2000-01-07-program.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Program"
+bg: white
+color: black
+---
+
+## Workshop Program
+
+
+0845 - 0900	&nbsp;&nbsp; **Loading Presentations to Computer**
+
+0900 - 0905	&nbsp;&nbsp; **Opening Remarks** 
+
+0905 - 0950 &nbsp;&nbsp; **Invited Talk 1 (Joel Nothman)**
+
+0950 - 1030 &nbsp;&nbsp; **Lightning Presentation for Posters 1** 
+
+1030 - 1100 &nbsp;&nbsp; **Break**
+
+1100 - 1145 &nbsp;&nbsp; **Poster Session 1**
+
+AllenNLP: A Deep Semantic Natural Language Processing Platform <br>
+*Matt Gardner, Joel Grus, Mark Neumann, Oyvind Tafjord, Pradeep Dasigi, Nelson Liu, Matthew Peters, Michael Schmitz and Luke Zettlemoyer*
+
+Stop Word Lists in Free Open-source Software Packages <br>
+*Joel Nothman, Hanmin Qin and Roman Yurchak*
+
+Texar: A Modularized, Versatile, and Extensible Toolbox for Deep Text Generation <br>
+*Zhiting Hu, Zichao Yang, Tiancheng Zhao, Haoran Shi, Junxian He, Di Wang, Xuezhe Ma, Zhengzhong Liu, Xiaodan Liang, Lianhui Qin, Devendra Singh Chaplot, Bowen Tan, Xingjiang Yu and Eric Xing*
+
+The ACL Anthology <br>
+*Daniel Gildea, Min-Yen Kan, Nitin Madnani, Christoph Teichmann and Martin Villalba*
+
+The risk of sub-optimal use of Open Source NLP Software: UKB is inadvertently state-of-the-art in knowledge-based WSD <br>
+*Eneko Agirre, Oier Lopez de Lacalle and Aitor Soroa*
+
+1200 - 1400 &nbsp;&nbsp; **Lunch**
+
+1400 - 1445 &nbsp;&nbsp; **Invited Talk 2 (Christopher Manning)** 
```

Comment:
Any particular reason why talks are ordered this way (Talk 2 -> Prof. Manning, Talk 3 -> Matt and Ines)? Should I ask them for their preferences?

Reply:
No particular reason for the order. But I think it'll be good to tell them and ask whether they're okay with it.